R/zv.R

```diff
@@ -7,6 +7,8 @@
 #' @param removals A character string that contains the names of
 #'  columns that should be removed. These values are not determined
 #'  until [prep.recipe()] is called.
+#' @param group An optional character string or call to [dplyr::vars()] that can be used to
+#'  specify a group(s) within which to identify variables that contain only a single value.
```

Comment:
Please make a note that any variables used here will be removed from `terms`.

Reply:
That is currently not happening. Are you saying that 

```
recipe(~ ., data = mtcars) %>%
  step_zv(all_predictors(), group = "am")
```

Should remove `am` as well as the zero variance columns?

Reply:
No, I mean that if the terms selector is something like `all_predictors()` and their grouping variable is within `all_predictors()`, the grouping variable will never be removed.